fitbenchmarking/controllers/bumps_controller.py

```diff
@@ -21,7 +21,12 @@ class BumpsController(Controller):
     """
 
     algorithm_check = {
-            'all': ['amoeba', 'lm-bumps', 'newton', 'de', 'scipy-leastsq'],
+            'all': ['amoeba',
+                    'lm-bumps',
+                    'newton',
+                    'de',
+                    'scipy-leastsq',
+                    'dream'],
             'ls': ['lm-bumps', 'scipy-leastsq'],
             'deriv_free': ['amoeba', 'de'],
             'general': ['amoeba', 'newton', 'de'],
```

Comment:
From what I can tell, dream is basically de bu with some allowance to not improve in each step -- I guess this helps it avoid suboptimal local minima. Can you add dream to these?

Reply:
Addressed in lastest commit